### DISCUSSION TITLE
Proposta de alteração na  forma de escolha da sede

### DISCUSSION THREAD

**Discussion by jamesperes** [apyb/comunidade#60] [Geral]
Olá Pessoal,
Gostaria de sugerir algumas alterações na escolha das sedes da Python Brasil, transformando elas mais dinâmicas e inclusivas.
 1 - Retirar do site dos associados que somente os associados podem votar e liberar para que todos presentes no evento possam votar.
2 - A apresentação das cidades candidatas sejam no primeiro dia de evento. (Sei que é um pouco difícil rs)
3 - Os votos podem ser em papel em uma urna onde cada kit na entrada tenha uma cédula de votação 
4 - Durante a assembleia é feita a contagem dos votos.

Gostaria muito a opinião de vcs sobre o assunto.
=)

**Comment by tiidadavena** [comment]
Achei bem interessante a ideia e considero o ponto 1 como **essencial** :D

Sobre o tópico 2, acho que aumenta o hype (usando terminologia dos jovens hehe) para submissão de cidade-sede; fomentando, assim, o compartilhamento de experiências durante o evento. Um possível resultado seriam mais submissões mais sólidas.

Já sobre os pontos 3 e 4, como que seria feita essa auditoria? Teria que eleger pessoas isentas para constituir uma Comissão Eleitoral (CE), para fiscalizar e acompanhar o processo. Creio que a diretoria, nesse caso, receberia os resultados e apoiaria as ações da Comissão Eleitoral.

**Comment by biancarosa** [comment]
Acredito que num modelo de evento hibrido, a gente caminha pra que pessoas presentes no evento estejam também online e é importante incluir essas pessoas no ponto 1. Minha sugestão é votar no periodo do evento, porém de forma online, similar a votacao da PSF para diretoria (que utiliza helios) e as pessoas podem votar digitalmente. Isso facilita os pontos 3 e 4.

**Reply by rougeth** [reply]
Esse é um tópico que vale uma discussão a parte. Esse ano fizemos o evento híbrido e a experiência foi no mínimo caótica hahaha. A org ainda está se reunindo e discutindo post-mortem, com o feedback delas, acho que devemos conversar sobre o modelo híbrido como um todo.

**Comment by rougeth** [comment]
Valeu por trazer esse tópico @jamesperes!

> 1 - Retirar do site dos associados que somente os associados podem votar e liberar para que todos presentes no evento possam votar.

A gente lista como um benefício para as pessoas associadas, mas na verdade, qualquer pessoa por participar. O que nós fazemos nas palestras e lightning talks da APyB, é dizer que a votação para cidade sede é aberta para qualquer participante do evento. @jamesperes se você não puder, eu abro um PR para essa mudança.

> 2 - A apresentação das cidades candidatas sejam no primeiro dia de evento. (Sei que é um pouco difícil rs)

Historicamente, a gente conta com o entusiasmo dos primeiros dias do evento para que as pessoas se motivem a assumir a próxima edição. A gente podia fazer um levantamento de quantas propostas foram para a Python Brasil já prontas (ou pelo menos decididas hahaha). Ribeirão Preto a ideia existia, mas não estava decidida, Manaus foi planejada. Floripa e BH (acho) se organizaram durante o evento. Natal não lembro.

> 3 - Os votos podem ser em papel em uma urna onde cada kit na entrada tenha uma cédula de votação
> 4 - Durante a assembleia é feita a contagem dos votos.

Acho que é um trabalho que a direção da APyB não tem condições de fazer hoje, considerando, no modelo atual, a baixa participação ativa da comunidade nas tarefas da associação. E o período próximo ao evento é o que mais demanda atenção da direção. 

Além disso, me parece nem compensar o trabalho, por exemplo, esse ano nem tivemos escolha, era o Rio ou nenhuma cidade sede.

Para a questão de mão de obra, uma opção seria a comunidade criar um grupo de trabalho focado na escolha da cidade sede (inclusive pensando em como incentivar mais submissões na assembleia).
